crates/core/src/formatting/writer.rs

```diff
@@ -1,9 +1,10 @@
 use bumpalo::Bump;
 
-use super::collections::{GraphNode, GraphNodeIterator};
-use super::print_items::WriterInfo;
-use super::StringContainer;
-use super::WriteItem;
+use super::{
+  collections::{GraphNode, GraphNodeIterator},
+  print_items::WriterInfo,
+  StringContainer, WriteItem,
+};
```

Comment:
I've been gradually switching the code over to have one `use` statement per import because we do that in Deno to try to reduce the pain of merge conflicts. Not a big deal though. I will update this repo to use the unstable "imports_granularity" rustfmt option https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#Item%5C%3A